src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -355,8 +355,8 @@ const UniqueTokenExpandedState = ({
   const isActionsEnabled = !external && !isReadOnlyWallet;
   const hasSendButton = isActionsEnabled && isSendable;
 
-  const hasEditButton =
-    isActionsEnabled && profilesEnabled && isENS && ensProfile.isOwner;
+  const isENSActionsEnabled = isActionsEnabled && profilesEnabled && isENS;
```

Comment:
maybe leave this as `hasEditButton`, and the below line could be `hasENSExtendDurationButton` or something

Reply:
done!